build.sh

```diff
@@ -153,12 +160,14 @@ get-couch-tarball() {
 }
 
 build-couch() {
+  # We will be changing user to 'jenkins' - ensure it has write permissions
+  chmod a+rwx pkgs pkgs/couch pkgs/js
   # $1 is plat, $2 is the optional path to a dist tarball
   docker run \
       --mount type=bind,src=${SCRIPTPATH},dst=/home/jenkins/couchdb-pkg \
       -w /home/jenkins/couchdb-pkg \
       couchdbdev/$1-erlang-${ERLANGVERSION} \
-      make copy-couch $1 COUCHTARBALL=${COUCHTARBALL}
+      make copy-couch $1 COUCHTARBALL=${COUCHTARBALL} PKGARCH=${PKGARCH}
```

Comment:
No, this doesn't work, sorry.

The architecture outside of the docker container may not match the architecture inside the container.

We're definitely going to end up having to run builds for s390x on non-s390x machines.

Sorry to throw a monkey wrench in your plans here.

Reply:
Understood.

I'll try to turn this around this evening.  Please stand-by.